packages/flutter_tools/lib/src/commands/multi_drive.dart

```diff
@@ -0,0 +1,375 @@
+import 'dart:async';
+import 'dart:convert' show JSON;
+import 'dart:io' as io;
+
+import 'package:path/path.dart' as path;
+import 'package:test/src/executable.dart' as executable; // ignore: implementation_imports
+
+import '../android/android_device.dart' show AndroidDevice;
+import '../application_package.dart';
+import '../base/file_system.dart';
+import '../base/common.dart';
+import '../base/logger.dart';
+import '../base/os.dart';
+import '../base/process.dart';
+import '../build_info.dart';
+import '../dart/sdk.dart';
+import '../device.dart';
+import '../globals.dart';
+import '../ios/simulators.dart' show SimControl, IOSSimulatorUtils;
+import '../run.dart';
+import '../runner/flutter_command.dart';
+import 'build_apk.dart' as build_apk;
+import 'run.dart';
+
+class MultiDriveCommand extends FlutterCommand {
+  MultiDriveCommand() {
+    addBuildModeFlags(defaultToRelease: false);
+
+    argParser.addFlag('trace-startup',
+        negatable: true,
+        defaultsTo: false,
+        help: 'Start tracing during startup.');
+
+    /*
+    argParser.addOption('route',
+        help: 'Which route to load when running the app.');
+    */
```

Comment:
Please remove the commented out block.
We might want to add the 'route' option in the spec file later.